src/options/didyoumean_test.cpp

```diff
@@ -751,6 +751,7 @@ set<string> getOptionStrings() {
       "no-sets-propagate",
       "sets-eager-lemmas",
       "no-sets-eager-lemmas",
+      "cube-test",
```

Comment:
You don't have to add `cube-test` here. This is just a test file and horribly outdated. We should probably remove the file in the future.